wolfhsm/wh_packet.h

```diff
@@ -29,11 +29,22 @@
     #define WOLFHSM_PACK
 #endif
 
+#ifdef WOLFHSM_PACK
+#undef WOLFHSM_PACK
+#define WOLFHSM_PACK
+#endif
+
+/*#pragma GCC diagnostic push
+#pragma GCC diagnostic error "-Wpadded"
+*/
```

Comment:
lets remove commented out code, I'm going to cook up a solution to this in a subsequent PR